### DISCUSSION TITLE
pesq package installation and compilation issue

### DISCUSSION THREAD

**Discussion by javajadi** [ludlows/python-pesq#40] [Q&A] [answered]
Hey Ludlows,
             I have been trying to use your PESQ wrapper to check the voice quality, unfortunately I am having some issues. Could you please help me out to resolve this issue?

**Comment by ludlows** [comment, marked as answer]
thanks for using `pesq`.

have you tried with the command `pip install pesq`.

if so, what errors have been raised?

**Reply by javajadi** [reply]
Yes tried the above command and having some issue due to not having the Visual studio tool builder. Now it is resolved.
